draft-ietf-httpbis-header-structure.md

```diff
@@ -389,7 +390,7 @@ Example-IntItemHeader: 5; foo=bar
 
 ### Integers {#integer}
 
-Integers have a range of −999,999,999,999,999 to 999,999,999,999,999 inclusive (i.e., up to fifteen digits, signed), for IEEE 754 compatibility ({{IEEE754}}).
+Integers have a range of âˆ’999,999,999,999,999 to 999,999,999,999,999 inclusive (i.e., up to fifteen digits, signed), for IEEE 754 compatibility ({{IEEE754}}).
```

Comment:
This looks like some non-ASCII slipped in. I'll send a PR to fix that.

Reply:
My bad. @mnot was too quick in merging :-). See also https://github.com/httpwg/http-extensions/pull/1105/.

Reply:
Fixed with #1108